### ISSUE TITLE
Support black 22.1.0

### ISSUE THREAD

**Issue by coignetp** [koxudaxi/datamodel-code-generator#700]
**Is your feature request related to a problem? Please describe.**
`black` released a new version 22.1.0 https://github.com/psf/black/releases/tag/22.1.0

**Describe the solution you'd like**
Being able to use `black` 22.1.0 and the latest `datamodel-code-generator` in the same project

**Describe alternatives you've considered**
Pin black to an earlier for now

**Additional context**
In particular, this new version changed `find_project_root ` return value used by `datamodel-code-generator` package.
* 22.1.0 https://github.com/psf/black/blob/d038a24ca200da9dacc1dcb05090c9e5b45b7869/src/black/files.py#L34
* 21.12b0 https://github.com/psf/black/blob/f1d4e742c91dd5179d742b0db9293c4472b765f8/src/black/files.py#L34
Using `black` 22.1.0 with `datamodel-code-generator` 0.11.17 causes this kind of error:
```
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/datamodel_code_generator/format.py", line 44, in __init__
    path = root / "pyproject.toml"
TypeError: unsupported operand type(s) for /: 'tuple' and 'str'
```

**Comment by alex9311** [comment]
for those looking for a workaround until #701  is merged, pinning to `black==21.12b0` works of course

**Comment by koxudaxi** [comment]
@coignetp cc: @alex9311 
I forgot to notify you.
Thank you very much 🙇 
https://github.com/koxudaxi/datamodel-code-generator/releases/tag/0.11.18
